### PR TITLE
Reassigned and updated font/font material on some interactable prefabs.

### PR DESCRIPTION
**Pull request by cre8ivepark**
## Overview
Some of the Interactable (button) prefabs had missing font or default font material. Updated/Re-assigned font and material.

## Changes
- Fixes: #5122

### CODE REVIEW

File: Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/ButtonHoloLens1.prefab [hunk 1]
```diff
@@ -388,9 +390,10 @@ MonoBehaviour:
   InputActionId: 0
   IsGlobal: 0
   Dimensions: 1
+  StartDimensionIndex: 0
   CanSelect: 1
   CanDeselect: 1
-  VoiceCommand: Button
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/ButtonHoloLens1.prefab]: Do we want this to keep its voice command?

File: Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/ButtonHoloLens1.prefab [hunk 2]
```diff
@@ -826,3 +831,211 @@ MeshRenderer:
   m_SortingLayerID: 0
   m_SortingLayer: 0
   m_SortingOrder: 0
+--- !u!1 &7299770270808978802
+GameObject:
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/ButtonHoloLens1.prefab]: Is this new GameObject intentional?

Comment (cre8ivepark) [reply Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/ButtonHoloLens1.prefab]: Yes, it was missing the text label. Added it back to make it consistent with HoloLens 2 buttons. 
I'll check the voice command string!

File: Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/ButtonHoloLens1.prefab (hunk 1 above)
Comment (cre8ivepark) [reply Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/ButtonHoloLens1.prefab]: Re-assigned the voice command.

**Comment by timGerken** [comment]
There are some other issues referenced in #5122 that this PR doesn't fix. See further comments there.

**Comment by wiwei** [comment]
@cre8tivepark, can you resolve the prefab conflict? We can then trigger CI and get this in.

We can fix the other latent issues in a separate PR as well.

**Pull request merged**